testutil/test_helpers.go

```diff
@@ -258,16 +277,20 @@ func (suite *IBCConnectionTestSuite) SetupCCVChannels() {
 func NewProviderConsumerCoordinator(t *testing.T) *ibctesting.Coordinator {
 	coordinator := ibctesting.NewCoordinator(t, 3)
 	chainID := ibctesting.GetChainID(1)
-	coordinator.Chains[chainID] = NewTestChain(t, coordinator, icssimapp.ProviderAppIniter, chainID)
+
+	ibctesting.DefaultTestingAppInit = icssimapp.ProviderAppIniter
+	coordinator.Chains[chainID] = ibctesting.NewTestChain(t, coordinator, chainID)
 	providerChain := coordinator.GetChain(chainID)
 
+	ibctesting.DefaultTestingAppInit = SetupTestingApp(cmttypes.TM2PB.ValidatorUpdates(providerChain.Vals))
 	chainID = ibctesting.GetChainID(2)
-	coordinator.Chains[chainID] = NewTestChainWithValSet(t, coordinator,
-		SetupTestingApp(), chainID, providerChain.Vals, providerChain.Signers)
+	coordinator.Chains[chainID] = ibctesting.NewTestChainWithValSet(t, coordinator,
+		chainID, providerChain.Vals, providerChain.Signers)
 
+	ibctesting.DefaultTestingAppInit = SetupTestingApp(cmttypes.TM2PB.ValidatorUpdates(providerChain.Vals))
```

Comment:
Is it a typo? why do we need the same statement two times? L285, L290

Reply:
Right, removed